docs/Guide.RunningOnCI.md

```diff
@@ -68,6 +68,8 @@ env:
   - NODE_VERSION=stable
 
 install:
+- brew tap wix/brew
+- brew install --HEAD applesimutils
```

Comment:
Please do no longer use `—HEAD`. There are proper versions now, which should be stable.

Reply:
Thank you! Updated the pull request.